### DISCUSSION TITLE
Community

### DISCUSSION THREAD

**Discussion by carlos-decastro** [randomcard/trinity#72] [Q&A]
Is there any community for us to exchange information about trinity continuum campaigns and modules created for the system? If yes and you prefer, send me on Discord Carlos Castro#9944
And congratulations on the excellent work with the system, it's perfect!

**Comment by randomcard** [comment]
Thank you! I'm glad it's working out!
There's no community specifically for this Foundry system, but in case you haven't seen them, there are:
**Onyx Path on Discord** (Trinity Continuum Channel): https://discordapp.com/channels/478634844532768768/534427048257650704
**Onyx Path / Trinity Continuum Forum**: http://forum.theonyxpath.com/forum/main-category/main-forum/trinity
**Foundry on Discord** (Other Systems Channel): https://discordapp.com/channels/170995199584108546/701846414208008302

Please let me know if you have any suggestions for the system. I haven't actually ran a game with this yet (I hope to soon), so I expect there's areas to improve still. And I'll send this to you via Discord as well. :)
